crawler/storage/elastic_storage.py

```diff
@@ -0,0 +1,120 @@
+# © Copyright 2021 HP Development Company, L.P.
+import os
+import zipfile
+import io
+from datetime import datetime
+from urllib.parse import urlparse
+from re import subn
+from elasticsearch import Elasticsearch, helpers
+
+from utils import SubCrawlColors, SubCrawlHelpers
+from .default_storage import DefaultStorage
+
+
+class ElasticStorage(DefaultStorage):
+
+    cfg = None
+    logger = None
+    es = None
+    index = None
+    archive_location = None
+    archive_content = False
+    max_fields = 0
+
+    def __init__(self, config, logger):
+        self.cfg = config
+        self.logger = logger
+        self.archive_location = SubCrawlHelpers.get_config(self.cfg,'elasticsearch', 'archive_log_location')
+        self.archive_content = SubCrawlHelpers.get_config(self.cfg, 'elasticsearch', 'archive_response_content')
+        self.index = SubCrawlHelpers.get_config(self.cfg, 'elasticsearch', 'index')
+
+        try:
+            self.es = Elasticsearch([{'host': SubCrawlHelpers.get_config(self.cfg, 'elasticsearch', 'host'),
+                                    'port': SubCrawlHelpers.get_config(self.cfg, 'elasticsearch', 'port'),
+                                    'index': self.index}])
+            self.es.ping()
+
+            if self.es.indices.exists(self.index) == False:
```

Comment:
[Programming recommendations say](https://www.python.org/dev/peps/pep-0008/#programming-recommendations): _Don't compare boolean values to True or False using ==:_

Therefore, I would change it to:
```
if not self.es.indices.exists(self.index):
```

Reply:
Fixed!